### ISSUE TITLE
195-visitors-analytics-with-goaccess

### ISSUE THREAD

**Issue by retifrav** [retifrav/retifrav.github.io#48]
[Visitors analytics with GoAccess](https://retifrav.github.io/blog/2020/05/20/visitors-analytics-with-goaccess/) comments.

**Comment by nkvname** [comment]
Thanks for the article! Is it possible to add IP addresses to "ignore-referer"?

**Comment by retifrav** [comment]
Yes, I think, it should not make any difference, after all it's just a string value in the header, so it should work the same way:

```
...
ignore-referer volcable.ru
ignore-referer 92.53.114.87
...
```
